packages/zero-runtime/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/zero-runtime",
-  "version": "0.0.1-alpha.0",
+  "version": "0.0.1-alpha.1",
```

Comment:
```suggestion
  "version": "0.0.1-alpha.0",
```

Shouldn't this be zero? It's the first version.

Reply:
Same for the other package

Reply:
Good point.

Reply:
Next time, we could consider dropping the `alpha` and starting with `0.1.0` according to [semver](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).